src/theme/variables/testDark.ts

```diff
@@ -122,9 +122,16 @@ export const testDark: Theme = {
   loadingIcon: palette.glowPurple,
 
   // Background
-  backgroundGradientColors: [palette.deepPurple, palette.darkPurple2],
-  backgroundGradientStart: { x: 0, y: 0 },
-  backgroundGradientEnd: { x: 0, y: 1 },
+  background: {
+    blurR: 100,
+    color: 'black',
+    dotOpacity: 0.4,
+    dots: [
+      { color: '#997833', cx: '45%', cy: '-20%', r: 100 }, // Mid-right
+      { color: '#999999', cx: '-40%', cy: '-40%', r: 90 }, // Top-left
+      { color: '#15cb7f', cx: '-65%', cy: '50%', r: 250 } // Bottom left
```

Comment:
Why not introduce these colors to the theme palette?

Reply:
Perhaps use the same or similar names from `edgeDark.ts`? If you have to add colors to those themes palettes, I say it's worth-while to keep things integrated for when design efforts are focused on these files.

Reply:
I added an extra commit for test theme files